eng/pipelines/dotnet-buildtools-prereqs-ubuntu16.yml

```diff
@@ -10,7 +10,9 @@ pr: none
 variables:
 - template: variables/common.yml
 - name: imageBuilder.pathArgs
-  value: src/ubuntu/16.04*
+  value: --path 'src/ubuntu/16.04*'
 
 stages:
-  - template: stages/build-repo.yml
+  - template: ../common/templates/stages/build-test-publish-repo.yml
+    parameters:
+      linuxAmdBuildJobTimeout: 120
```

Comment:
I am surprised more legs didn't require a longer timeout. (e.g. the ubuntu 14.04 leg)

Reply:
Ubuntu 14.04 actually runs pretty quick: ~30 mins.  I did however determine that CentOS and openSUSE needed to have their timeouts extended.